src/hooks/useArticleTextSize.js

```diff
@@ -1,12 +1,13 @@
-import { useState, useLayoutEffect } from 'preact/hooks'
+import { useEffect, useContext } from 'preact/hooks'
+import { FontContext } from 'contexts'
 import { articleTextSize } from 'utils'
 
-export const useArticleTextSize = (dependencies = []) => {
-  const [textSize, setTextSize] = useState(articleTextSize.get())
+export const useArticleTextSize = () => {
+  const { fontSizeClass, setFontSizeClass } = useContext(FontContext)
 
-  useLayoutEffect(() => {
-    articleTextSize.init()
-  }, dependencies)
+  useEffect(() => {
+    setFontSizeClass(articleTextSize.getFontSizeClassName())
+  }, [fontSizeClass])
 
-  return [textSize, setTextSize]
+  return [fontSizeClass, setFontSizeClass]
 }

```

Comment:
this hook can be fully removed as it does return the same thing from `useContext`, and the font size class name is being modified in `app.js`

Reply:
I see... so just using the `FontContext` directly, I think you're right, let me try that

Reply:
So check out https://github.com/wikimedia/wikipedia-kaios/pull/300/commits/592781eabb5107738f2b1fbb975443313d505c27. I think the only difference I'm noticing is that useArticleLinksNavigation is _slightly_ off now. Is it just me or do you notice that as well? Looking at `fontSizeClass` in Article.js and App.js, they seem to be updating as expected. But maybe there's a discrepancy with the `fontSizeClass` dependency in Article.js.

Haven't removed useArticleTextSize completely yet for testing purposes